src/safe_apps/tests/test_signals.py

```diff
@@ -1,11 +1,12 @@
 import responses
 from django.test import TestCase, override_settings
 
-from safe_apps.models import SafeApp, Tag
-from safe_apps.tests.factories import ProviderFactory
+from ..models import SafeApp, Tag
+from ..tests.factories import ProviderFactory
 
 
 @override_settings(
+    FF_HOOK_EVENTS=False,
```

Comment:
Should we add this `FF_HOOK_EVENTS` setting to the other TestCases also? (`ProviderHookTestCase`, `TagHookTestCase`)? I\ understand it should be `false` anyway because it's the default value fo the setting, but I think it would be nice to add it as you did for the other TestCases, for the sake of consistency.

Reply:
We should! Thanks for pointing that out. We should not assume a default for the tests in this case 👍

Reply:
Addressed in [bfcbbbf](https://github.com/safe-global/safe-config-service/pull/964/commits/bfcbbbfc65b52d790114b3960c1e3dce3b2e30be)